scripts/dts/edtlib.py

```diff
@@ -724,9 +724,19 @@ def unit_addr(self):
 
         addr = _translate(addr, self._node)
 
-        if self.regs and self.regs[0].addr != addr:
-            self.edt._warn("unit-address and first reg (0x{:x}) don't match "
-                           "for {}".format(self.regs[0].addr, self.name))
+        try:
+            EXTRA_DTC_FLAGS = os.environ.get("EXTRA_DTC_FLAGS")
+            if EXTRA_DTC_FLAGS is None:
+                EXTRA_DTC_FLAGS = ""
+        except:
+            EXTRA_DTC_FLAGS = ""
```

Comment:
Can shorten this down to this:

```python
extra_dtc_flags = os.getenv("EXTRA_DTC_FLAGS", "")
```

`""` is the fallback value.

The `except` would never hit, because `get()` never raises.